dags/common/utils.py

```diff
@@ -49,3 +49,10 @@ def check_value(value):
             return all(value)
         return False
     return False
+
+
+def parse_to_int(value, logger):
+    try:
+        return [int(value)]
```

Comment:
the array logic shouldn't be part of this function, since it's related to the parser.

Reply:
I know, but it's the same situation as with dois, that we discussed

Reply:
Hmm, what do you mean? It's a new function we can add it to the extraction function

Reply:
I understood it wrong